api/src/treestatus_api/static/ui/main-c7f8ce4c8d3beb7f7619.js

```diff
@@ -1 +1 @@
y=Jt.tfoot=Jt.colgroup=Jt.caption=Jt.thead,Jt.th=Jt.td;var Xt=/<|&#?\w+;/;!function(){var t=at.createDocumentFragment(),e=t.appendChild(at.createElement("div")),r=at.createElement("input");r.setAttribute("type","radio"),r.setAttribute("checked","checked"),r.setAttribute("name","t"),e.appendChild(r),vt.checkClone=e.cloneNode(!0).cloneNode(!0).lastChild.checked,e.innerHTML="<textarea>x</textarea>",vt.noCloneChecked=!!e.cloneNode(!0).lastChild.defaultValue}();var Zt=at.documentElement,te=/^key/,ee=/^(?:mouse|pointer|contextmenu|drag|drop)|click/,re=/^([^.]*)(?:\.(.+)|)/;mt.event={global:{},add:function(t,e,r,n,o){var i,a,c,s,u,l,f,d,h,_,p,g=Mt.get(t);if(g)for(r.handler&&(i=r,r=i.handler,o=i.selector),o&&mt.find.matchesSelector(Zt,o),r.guid||(r.guid=mt.guid++),(s=g.events)||(s=g.events={}),(a=g.handle)||(a=g.handle=function(e){return void 0!==mt&&mt.event.triggered!==e.type?mt.event.dispatch.apply(t,arguments):void 0}),e=(e||"").match(It)||[""],u=e.length;u--;)c=re.exec(e[u])||[],h=p=c[1],_=(c[2]||"").split(".").sort(),h&&(f=mt.event.special[h]||{},h=(o?f.delegateType:f.bindType)||h,f=mt.event.special[h]||{},l=mt.extend({type:h,origType:p,data:n,handler:r,guid:r.guid,selector:o,needsContext:o&&mt.expr.match.needsContext.test(o),namespace:_.join(".")},i),(d=s[h])||(d=s[h]=[],d.delegateCount=0,f.setup&&!1!==f.setup.call(t,n,_,a)||t.addEventListener&&t.addEventListener(h,a)),f.add&&(f.add.call(t,l),l.handler.guid||(l.handler.guid=r.guid)),o?d.splice(d.delegateCount++,0,l):d.push(l),mt.event.global[h]=!0)},remove:function(t,e,r,n,o){var i,a,c,s,u,l,f,d,h,_,p,g=Mt.hasData(t)&&Mt.get(t);if(g&&(s=g.events)){for(e=(e||"").match(It)||[""],u=e.length;u--;)if(c=re.exec(e[u])||[],h=p=c[1],_=(c[2]||"").split(".").sort(),h){for(f=mt.event.special[h]||{},h=(n?f.delegateType:f.bindType)||h,d=s[h]||[],c=c[2]&&new RegExp("(^|\\.)"+_.join("\\.(?:.*\\.|)")+"(\\.|$)"),a=i=d.length;i--;)l=d[i],!o&&p!==l.origType||r&&r.guid!==l.guid||c&&!c.test(l.namespace)||n&&n!==l.selector&&("**"!==n||!l.selector)||(d.splice(i,1),l.selector&&d.delegateCount--,f.remove&&f.remove.call(t,l));a&&!d.length&&(f.teardown&&!1!==f.teardown.call(t,_,g.handle)||mt.removeEvent(t,h,g.handle),delete s[h])}else for(h in s)mt.event.remove(t,h+e[u],r,n,!0);mt.isEmptyObject(s)&&Mt.remove(t,"handle events")}},dispatch:function(t){var e,r,n,o,i,a,c=mt.event.fix(t),s=new Array(arguments.length),u=(Mt.get(this,"events")||{})[c.type]||[],l=mt.event.special[c.type]||{};for(s[0]=c,e=1;e<arguments.length;e++)s[e]=arguments[e];if(c.delegateTarget=this,!l.preDispatch||!1!==l.preDispatch.call(this,c)){for(a=mt.event.handlers.call(this,c,u),e=0;(o=a[e++])&&!c.isPropagationStopped();)for(c.currentTarget=o.elem,r=0;(i=o.handlers[r++])&&!c.isImmediatePropagationStopped();)c.rnamespace&&!c.rnamespace.test(i.namespace)||(c.handleObj=i,c.data=i.data,void 0!==(n=((mt.event.special[i.origType]||{}).handle||i.handler).apply(o.elem,s))&&!1===(c.result=n)&&(c.preventDefault(),c.stopPropagation()));return l.postDispatch&&l.post
```

Comment:
Is this supposed to be human readable?

Reply:
I saw the previous edit was to this file and did a grep for esr78, ensure that we have esr78 and esr91